WordPress/src/main/res/layout/login_form_screen.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <include
+        android:id="@+id/toolbar"
+        layout="@layout/toolbar_login" />
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@+id/toolbar"
+        android:layout_above="@+id/bottom_buttons"
+        android:fillViewport="true">
+
+        <ViewStub
+            android:id="@+id/login_form_content_stub"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_vertical"
+            android:layout_marginTop="@dimen/margin_extra_large"
+            android:layout_marginBottom="@dimen/margin_extra_large"/>
+    </ScrollView>
+
+    <RelativeLayout
```

Comment:
Good call. I will fix this in the PR the bottom buttons are primarily introduced. (That'll be https://github.com/wordpress-mobile/WordPress-Android/pull/6230)